Neos.Utility.MediaTypes/Scripts/updatemediatypes.php

```diff
@@ -31,6 +31,9 @@
     }
 }
 
+// Add non-registered mime types that we still want to support:
+$mediaTypesAndFileExtensions['application/vnd.apple.pkpass'] = ['pkpass'];
+
```

Comment:
is this script run automatically or should it be done manually and included with this PR?

Reply:
I see nothing on Jenkins